readme.md

```diff
@@ -78,7 +78,7 @@ Spelling - readme.md
   peeling
 ```
 
-Where `speling` will be highlighted in red.
+Where `spelling` will be highlighted in red.
```

Comment:
I think at least this misspelling is intentional.  `spelling` would not be highlighted in red since it is spelled properly.